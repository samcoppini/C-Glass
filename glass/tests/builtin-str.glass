{M
    [m
        (_s)S!
        (_o)(Output)!

        "Testing S.l"(_o)(ol).?
        "This is 26 characters long"(_s)l.?(_o)(onl).?

        "Testing S.i"(_o)(ol).?
        "ABCDEF"<0>(_s)i.?(_o)(ol).?
        "ABCDEF"<3>(_s)i.?(_o)(ol).?

        "Testing S.si"(_o)(ol).?
        "Words"<1>"a"(_s)(si).?(_o)(ol).?

        "Testing S.a"(_o)(ol).?
        "Hello ""World"(_s)a.?(_o)(ol).?

        "Testing S.d"(_o)(ol).?
        "Underground"<5>(_s)d.?(_o)(ol).?(_o)(ol).?

        "Testing S.e"(_o)(ol).?
        "Foo""Fooo"(_s)e.?(_o)(onl).?
        "Foo""Foo"(_s)e.?(_o)(onl).?

        "Testing S.ns"(_o)(ol).?
        <33>(_s)(ns).?(_o)(ol).?

        "Testing S.sn"(_o)(ol).?
        "!"(_s)(sn).?(_o)(onl).?
    ]
}
