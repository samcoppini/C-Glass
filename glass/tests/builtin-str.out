Testing S.l
26
Testing S.i
A
D
Testing S.si
Wards
Testing S.a
Hello World
Testing S.d
ground
Under
Testing S.e
0
1
Testing S.ns
!
Testing S.sn
33
